data/config/schema.xml

```diff
@@ -213,6 +213,15 @@ to save the current settings to XML.
 		<short>Word Collation</short>
 		<long>Words that should be ignored when sorting.</long>
 	</entry>
+ 
+	<entry name="game/notegraphscalingmode" type="int" value="0">
+		<limits>
+			<enum>Dynamic</enum>
+			<enum>Fix</enum>
```

Comment:
@Baklap4 should I change the label or needs it still a decision?

If I should change it I will do it today.

Reply:
Go ahead make it 'Fixed'

Reply:
Hmm, the class name and file names uses fix instead of fixed too. I think this should be changed to fixed.

'fix' is german. Sorry. Should be corrected IMO.

@Baklap4 ?

Reply:
Oo good catch! That should be corrected yes 👍

Reply:
Pushed and checks passed.